plugins/modules/ipagroup.py

```diff
@@ -400,6 +474,18 @@ def main():
                                              "group": group,
                                          }])
 
+                    if has_add_membermanager:
+                        # Remove membermanager users and groups
+                        if len(membermanager_user) > 0 or \
+                           len(membermanager_group) > 0:
```

Comment:
These lines show the same issue as before, just add `state: absent` to the test playbook.

I'd suggest the same fix here.